log-viewer/modules/parsers/TreeParser.ts

```diff
@@ -126,6 +130,22 @@ export class TimedNode extends LogLine {
     this.cpuType = cpuType;
   }
 
+  getBreadcrumbText() {
```

Comment:
I guess this should be abstract since `TimedNode` is to be extended? In fact should `TimedNode` be abstract too?

```
abstract getBreadcrumbText(): string
```

```
export abstract class TimedNode extends LogLine {
```

Reply:
TimedNode is not abstract (hence it cannot contain abstract methods).

Reply:
I know, I suggested making TimedNode abstract too 😉 
Even if you do not declare the class abstract you can also make the function optional I found recently. e.g
`getBreadcrumbText?(): string;`

Reply:
Why would we make the method optional when we want an answer? (from a concrete TimedNode instance)